src/api-spec/openapi/swagger/trade2.ts

```diff
@@ -12,28 +13,24 @@ export class TradeServiceHttp implements TraderClientInterface {
 
   async completeTrade(swapCompleteSerialized: Uint8Array): Promise<string> {
     const path = `/v1/trade/complete`;
-    const response = await fetch(this.providerUrl + path, {
-      method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify(swapCompleteSerialized),
+    const response = await axios.post(this.providerUrl + path, {
+     swapCompleteSerialized,
     })
-    const str = await response.text()
+    const str = await response.data
     console.log('completeTrade', str);
     return str
   }
 
   async proposeTrade(market: Market, tradeType: TradeType, swapRequestSerialized: Uint8Array): Promise<Uint8Array> {
     const path = `/v1/trade/propose`;
-    const response = await fetch(this.providerUrl + path, {
-      method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({
+    console.log('this.providerUrl', this.providerUrl);
+    const response = await axios.post(this.providerUrl + path, {
         market: market,
         type: tradeType.toString(),
-        swapRequest: SwapRequest.fromBinary(swapRequestSerialized),
-      }),
-    })
-    const str = await response.json()
+        swapRequest: SwapRequest.toJsonString(SwapRequest.fromBinary(swapRequestSerialized)),
+        //swapRequest: Buffer.from(swapRequestSerialized).toString('base64'),
```

Comment:
don think this work either:

we should pass each field of the swap request, one by one, for the type that swagger defintion specify.

Only thing to keep into consideration that `bytes` fields are passed as `string` in JSON, which means base64

Reply:
```json
{
  "market": {
    "base_asset": "Hello",
    "quote_asset": "Hello"
  },
  "type": 0,
  "swap_request": {
    "id": "ec033502-fbe2-4ab6-a455-f5711aa749ba",
    "amount_p": 100,
    "asset_p": "Hello",
    "amount_r": 100,
    "asset_r": "Hello",
    "transaction": "Hello",
    "input_blinding_key": {
      "script": "base64BlindingKey" 
    },
    "output_blinding_key": {
      "script": "base64BlindingKey" 
    }
  }
}
```

Reply:
In general, though, this test must be dropped.

We should just write a new one in `test/trade.integration.test.ts` where we don't use the default client, but the new one defined in `openapi/swagger`.